Source/Public/New-ADOPSServiceConnection.ps1

```diff
@@ -0,0 +1,91 @@
+function New-ADOPSServiceConnection {
+  [cmdletbinding()]
+  param(
+      [parameter(Mandatory = $true)]
+      [string]$TenantId,
+
+      [parameter(Mandatory = $true)]
+      [string]$SubscriptionName,
+
+      [parameter(Mandatory = $true)]
+      [string]$SubscriptionId,
+
+      [parameter(Mandatory = $true)]
+      [string]$ServicePrincipalId,
+
+      [parameter(Mandatory = $true)]
+      [ValidateNotNullorEmpty()]
+      [string]$Project,
+
+      [parameter(Mandatory = $false)]
```

Comment:
Mandatory = $false is not needed here. Just remove it totally and replace with `[parameter()]` instead.